charts/s1-agent/templates/helper/statefulset.yaml

```diff
@@ -91,7 +91,7 @@ spec:
         {{- end }}
 {{- end }}
 {{- if or .Values.configuration.env.injection.enabled .Values.configuration.env.helper.communicator_enabled }}
-        {{- if and (include "site_key.secret.create" .)  (include "site_key.secret.name" .) }}
+        {{- if (include "site_key.secret.name" .) }}
```

Comment:
Just to confirm, we're removing the check for the existence of `.Values.secrets.site_key.value` because it's not necessary?
Or am I missing something?